server/routes.go

```diff
@@ -915,8 +916,61 @@ func (s *Server) GenerateRoutes() http.Handler {
 		)
 	}
 
-	r := gin.Default()
```

Comment:
`gin.Default()` attaches `gin.Recovery` and the default logging middleware. To prevent duplicate logs for the same request, use `gin.New()` instead